core/commands/pin.go

```diff
@@ -94,9 +94,9 @@ on disk.
 
 var rmPinCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Unpin an object from local storage",
+		Tagline: "Recursively unpin an object from local storage",
 		ShortDescription: `
-Removes the pin from the given object allowing it to be garbage
+Recursively removes the pin from the given object allowing it to be garbage
```

Comment:
(this is because is should not be able to turn `--recursive=false` on a command that `"Recursively unpins an object from local storage"`